lib/ui/onboarding/usecases/create_account_use_case.dart

```diff
@@ -7,44 +6,36 @@ import 'package:hypha_wallet/ui/architecture/result/result.dart';
 import 'package:hypha_wallet/ui/blocs/deeplink/deeplink_bloc.dart';
 import 'package:image_picker/image_picker.dart';
 
-class CreateAccountUseCase extends InputUseCase<Result<bool, HyphaError>, Input> {
+class CreateAccountUseCase extends InputUseCase<Result<bool, HyphaError>, CreateAccountInput> {
   final AuthRepository _authRepository;
 
   CreateAccountUseCase(this._authRepository);
 
   @override
-  Future<Result<bool, HyphaError>> run(Input input) async {
-    Get.showSnackbar(
-      const GetSnackBar(
-        message: 'NIK: Here is where you call create account using profile service',
-        duration: Duration(seconds: 3),
-      ),
-    );
-    return Result.error(HyphaError.api('Error creating account'));
-
+  Future<Result<bool, HyphaError>> run(CreateAccountInput input) async {
     try {
-      final bool result = await _authRepository.createUserAccount(
-        userName: input.userName,
+      final result = await _authRepository.createUserAccount(
         accountName: input.accountName,
-        image: input.image,
         userAuthData: input.userAuthData,
         inviteLinkData: input.inviteLinkData,
+        userName: input.userName,
+        image: input.image,
       );
       return Result.value(result);
-    } catch (e) {
-      return Result.error(HyphaError.api('Error creating account'));
```

Comment:
Comment: It's a little strange HyphaError doesn't wrap the OG error - I think it should. ?!